File: minesweeper-api/src/main/java/com/deviget/minesweeperapi/model/responseDto/MoveResponse.java
package com.deviget.minesweeperapi.model.responseDto;

import lombok.Data;

import java.util.List;

@Data
public class MoveResponse {
    private long gameId;
    private String gameStatus;
    private CellResponse[][] gameField;
    private String[] gameFieldView;
    private long gameTime;

}

File: MinesWeeper.APIClientLibrary/MinesWeeper.APIClientLibrary/Model/MoveResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MinesWeeper.APIClientLibrary.Model
{
    public class MoveResponse
    {
        public long gameId { set; get; }
        public string gameStatus { set; get; }
        public CellResponse[][] gameField { set; get; }
        public string[] gameFieldView { set; get; }
        public long gameTime { set; get; }
    }
}
